src/test/java/com/google/cloud/solutions/autotokenize/pipeline/DlpInspectionPipelineIT.java

```diff
@@ -314,14 +307,16 @@ public void makeOptions() throws IOException, ManagedProcessException {
       case JDBC_TABLE:
         var initScript = configParameters.get("initScript");
         var testDatabaseName = "test_" + new Random().nextLong();
-        testDBInstance.createDB(testDatabaseName);
-        testDBInstance.source(initScript, testDatabaseName);
+        databaseContainer = new MySQLContainer(DockerImageName.parse("mysql:8.0.24"));
```

Comment:
since the proposal is to use `@ClassRule` may be this can be changed to instead just create a new Database on the SQL container.